Loop/Views/BolusEntryView.swift

```diff
@@ -477,3 +518,14 @@ struct LabelBackground: ViewModifier {
             )
     }
 }
+
+extension Binding {
+    func onChange(_ handler: @escaping (Value) -> Void) -> Binding<Value> {
```

Comment:
This function will be build-in in iOS 14, but isn't currently available in iOS 13